### ISSUE TITLE
mark_messages_as_read applies only to messages (t4) and not comments (t1)

### ISSUE THREAD

**Issue by waglik** [not-an-aardvark/snoowrap#28]
Hi.

at the moment 

r.mark_messages_as_read(['51shsd', '51shxv'])

will only work if those are IDs of private messages. If I would like to mark as read messages that were comments to my users comments it would not work. Source code indicates why is that 

>const message_ids = messages.map(message => add_fullname_prefix(message, 't4_'));

My question is , would it be possible to customize this behavior a bit? Or is there another way to mark comment as read?

**Comment by not-an-aardvark** [comment]
This should be fixed in v1.2.0, which I'll publish to npm in a few minutes. To pass a comment to `mark_messages_as_read`, you can either prefix its ID with `t1_`, e.g. 

```js
r.mark_messages_as_read(['t1_abcdef'])
```

...or simply pass a comment object directly, e.g. 

```js
r.mark_messages_as_read([r.get_comment('abcdef')])
```

In the first case, the prefix is necessary because snoowrap needs to know that the ID is referring to a comment as opposed to a private message.

**Comment by waglik** [comment]
Works great! Thank you.
